docs/source/guides/user/chapters/configuring.rst

```diff
@@ -232,10 +229,6 @@ Example using the command-line:
 
   $ avocado run --verbose /bin/true
 
-.. note:: Currently we still have some "old style boolean" options where you
-  should pass "on" or "off" on the command-line. i.e: ``--sysinfo=off``. Those
-  options are going to be replaced soon.
-
```

Comment:
I think that is worthed to keep the message, or modified version, since that we still have a few other options to migrate, right?